--- xlock/xlock.c.setgid	2008-05-22 10:50:43.000000000 -0600
+++ xlock/xlock.c	2008-05-22 10:53:50.000000000 -0600
@@ -3343,6 +3343,7 @@ main(int argc, char **argv)
 /* that needs root rights (pam_unix e.g.) */
 #else
 
+#if 0
 #ifdef USE_VTLOCK
 	/* In order to lock VT switch we must issue an ioctl on console */
 	/* (VT_LOCKSWITCH). This ioctl MUST be issued by root. */
@@ -3357,6 +3358,7 @@ main(int argc, char **argv)
 	}
 #endif
 #endif
+#endif
 
 #ifdef SYNCHRONIZE
 	/* synchronize -- so I am aware of errors immediately */
